### PR TITLE
Fix https mixed content in footer

### PR DESCRIPTION
**Pull request by pior**

### CODE REVIEW

File: mtlpy/templates/inc/footer.html [hunk 1]
```diff
@@ -85,8 +84,7 @@
       }(document, 'script', 'facebook-jssdk'));
     </script>
     {% if GOOGLE_ANALYTICS %}
-    <script src="http://www.google-analytics.com/urchin.js" type="text/javascript">
-    </script>
+    <script src="https://www.google-analytics.com/urchin.js" type="text/javascript"></script>
```
Comment (mlhamel) [inline mtlpy/templates/inc/footer.html]: this remind me that we'll also need to migrate to ssl only when switching to heroku !

Comment (pior) [inline mtlpy/templates/inc/footer.html]: I don't know if we want to force HTTPS?
Maybe we could just spread https links and keep http working.

Comment (mlhamel) [inline mtlpy/templates/inc/footer.html]: dunno, i only thing we should definitely advertise the https one

Comment (pior) [inline mtlpy/templates/inc/footer.html]: Definitely

**Pull request merged**